org.skytemple.SkyTemple.yml

```diff
@@ -0,0 +1,144 @@
+app-id: org.skytemple.SkyTemple
+runtime: org.gnome.Platform
+runtime-version: '42'
+sdk: org.gnome.Sdk
+command: /app/bin/run.sh
+finish-args:
+  - "--share=network"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--filesystem=host"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-nightly
+cleanup:  # todo
+  - /bin/pcap-config
+  - /include
+  - /lib/libpcap.a
+  - /lib/pkgconfig
+  - /share/man
+modules:
+  - name: armips
+    buildsystem: simple
+    build-commands:
+      - mkdir -p build
+      - patch Core/ExpressionFunctions.cpp < armips.patch
+      - cd build && cmake -DCMAKE_BUILD_TYPE=Release ..
+      - cd build && make armips-bin
+      - install -Dm755 build/armips /app/bin/armips
+    sources:
+      - type: archive
+        url: https://github.com/Kingcom/armips/archive/v0.11.0.tar.gz
+        sha256: c94e29dfda3bdf853590d825562b9c73a3d8e8e995555e021c6b2a6451572681
+      - type: file
+        path: patches/armips.patch
+  - name: skytemple
```

Comment:
this is the main module, isnt't it ? it should be the last... (and merged with the "meta" one)